docs/graphql-admin/locales.mdx

```diff
@@ -0,0 +1,256 @@
+# GraphQL Admin: Locale overview
+
+## Introduction
+
+The BigCommerce GraphQL Admin API provides the same access to the locales field as the REST Management [Store Locale](/docs/rest-management/settings/store-locale) API. Having access to locale data in the graph alongside related data makes it easier to build headless storefronts.
```

Comment:
Let's avoid cross linking to this API, as we are trying to replace this old one with this new GQL. 
Let's just focus on this API being a way for developers to manage languages on channels. 

We should also call out that this release is available on Catalyst storefronts only for now.